app/validators/idv/in_person/form_address_validator.rb

```diff
@@ -5,8 +5,10 @@ module FormAddressValidator
       include Idv::FormAddressValidator
 
       included do
-        validates :same_address_as_id,
-                  presence: true
+        unless IdentityConfig.store.in_person_capture_secondary_id_enabled
+          validates :same_address_as_id,
+                    presence: true
+        end
```

Comment:
This should use [conditional validation](https://guides.rubyonrails.org/active_record_validations.html#using-a-symbol-with-if-and-unless), esp. since the value will later be determined per-user rather than globally.

Reply:
I wrote a test demonstrating how this causes a bug, and then I fixed the bug.